bigbluebutton-tests/playwright/parameters/parameters.spec.js

```diff
@@ -479,14 +479,58 @@ test.describe.parallel('Custom Parameters', { tag: '@ci' }, () => {
     });
   });
 
-  test.describe.parallel('Presentation', () => {
+  test.describe.parallel('Hide Presentation On Join Edge Cases', () => {
     test('Hide Presentation on join', async ({ browser, context, page }) => {
       const customParam = new CustomParameters(browser, context);
       await customParam.initModPage(page, true, { joinParameter: c.hidePresentationOnJoin });
       await customParam.initUserPage(true, context, { useModMeetingId: true, joinParameter: c.hidePresentationOnJoin });
       await customParam.hidePresentationOnJoin();
     });
 
+    test('Hide Presentation on join Screenshare', async ({ browser, context, page }) => {
+      const customParam = new CustomParameters(browser, context);
+      await customParam.initModPage(page, true, { joinParameter: c.hidePresentationOnJoin });
+      await customParam.initUserPage(true, context, { useModMeetingId: true, joinParameter: c.hidePresentationOnJoin });
+      await customParam.hidePresentationOnJoinScreenshare();
+    });
+
+    test('Hide presentation on join External video', { tag: '@flaky' }, async({ browser, context, page }) => {
```

Comment:
add a comment with the flaky reason + use `linkIusse(21589)` (just like https://github.com/antonbsa/bigbluebutton/blob/v3.0.x-release/bigbluebutton-tests/playwright/screenshare/screenshare.spec.js#L17-L18)